packages/venia-ui/lib/components/Checkbox/checkbox.css

```diff
@@ -44,12 +44,17 @@
     cursor: default;
 }
 
-.input:checked + .icon {
+/* When the input is disabled, update the cursor on the sibling label element. */
+.input:disabled ~ .label {
+    cursor: default;
+}
+
+.input:checked:enabled + .icon {
     --stroke: var(--venia-brand-color-1-700);
 }
 
-.input:active,
-.input:focus {
+.input:active:enabled,
+.input:focus:enabled {
```

Comment:
I added these `:enabled` and `:disabled` pseudo-classes here to more distinctly visually differentiate an enabled checkbox from a disabled one.

Previously, a disabled checkbox still looked very much like an enabled one, you just couldn't click it. It felt more like a bug.

Reply:
Awesome, this was definitely needed. Cursor only treatment doesn't mean much on mobile.